app/templates/views/dashboard/dashboard.html

```diff
@@ -29,10 +29,57 @@ <h2 class="font-body-xl margin-0">
 
     {{ ajax_block(partials, updates_url, 'template-statistics') }}
 
+    <h2 class="margin-top-4 margin-bottom-1">Batched Jobs</h2>
+    {% if notifications %}
+    <div class='job-table'>
+    {% endif %}
+      {% call(item, row_number) list_table(
+        notifications,
+        caption="Batched Jobs",
+        caption_visible=False,
+        border_visible=True,
+        empty_message='No batched job messages found &thinsp;(messages are kept for {} days)'.format(service_data_retention_days)|safe,
+        field_headings=['Template Name','Date/Time', 'Download (CSV) Report'],
+        field_headings_visible=False
+      ) %}
+        {% if item.job.original_file_name and item.job.id %}
+          {% call row_heading() %}
+            <a class="usa-link file-list-filename" href="/services/{{ item.service }}/jobs/{{ item.job.id }}">{{ item.job.original_file_name|replace('.csv', '') if item.job.id else '' }}</a>
+          {% endcall %}
+          {% call row_heading() %}
+            {{ item.status|format_notification_status_as_time(
+              item.created_at|format_datetime_short,
+              (item.updated_at or item.created_at)|format_datetime_short)
+            }}
+          {% endcall %}
+          {% call row_heading() %}
+            {% set availability = download_availability|selectattr('job_id', 'equalto', item.job.id)|first %}
+              {% if availability and availability.time_left != "Data no longer available" %}
+                <a class="usa-link file-list-filename" href="{{ availability.download_link[0] }}">{{ "Download" if item.job.original_file_name else '' }}</a>
+                <span>- {{ availability.time_left }}</span>
+              {% elif availability %}
+                <span>{{ availability.time_left }}</span>
+              {% endif %}
+          {% endcall %}
+        {% endif %}
+      {% endcall %}
+    {% if notifications %}
+      </div>
+    {% endif %}
```

Comment:
Should the call to the `list_table` component/macro also be inside of the `{% if notifications %}` check since it lives inside the `div` that would be rendered if that check is `True`?  If so, like this?

```suggestion
    {% if notifications %}
    <div class='job-table'>
      {% call(item, row_number) list_table(
        notifications,
        caption="Batched Jobs",
        caption_visible=False,
        border_visible=True,
        empty_message='No batched job messages found &thinsp;(messages are kept for {} days)'.format(service_data_retention_days)|safe,
        field_headings=['Template Name','Date/Time', 'Download (CSV) Report'],
        field_headings_visible=False
      ) %}
        {% if item.job.original_file_name and item.job.id %}
          {% call row_heading() %}
            <a class="usa-link file-list-filename" href="/services/{{ item.service }}/jobs/{{ item.job.id }}">{{ item.job.original_file_name|replace('.csv', '') if item.job.id else '' }}</a>
          {% endcall %}
          {% call row_heading() %}
            {{ item.status|format_notification_status_as_time(
              item.created_at|format_datetime_short,
              (item.updated_at or item.created_at)|format_datetime_short)
            }}
          {% endcall %}
          {% call row_heading() %}
            {% set availability = download_availability|selectattr('job_id', 'equalto', item.job.id)|first %}
              {% if availability and availability.time_left != "Data no longer available" %}
                <a class="usa-link file-list-filename" href="{{ availability.download_link[0] }}">{{ "Download" if item.job.original_file_name else '' }}</a>
                <span>- {{ availability.time_left }}</span>
              {% elif availability %}
                <span>{{ availability.time_left }}</span>
              {% endif %}
          {% endcall %}
        {% endif %}
      {% endcall %}
      </div>
    {% endif %}
```